website/docs/plugins/nevergrad_sweeper.md

```diff
@@ -0,0 +1,169 @@
+[HYDRA] Sweep output dir: multirun/2020-03-04/17-53-29
+[HYDRA] Launching 10 jobs locally
+[HYDRA] 	#0 : db=mnist batch_size=8 lr=0.032 dropout=0.5
+[__main__][INFO] - dummy_training(dropout=0.500, lr=0.032, db=mnist, batch_size=8) = 5.258
+[HYDRA] 	#1 : db=mnist batch_size=16 lr=0.035 dropout=0.714
+[__main__][INFO] - dummy_training(dropout=0.714, lr=0.035, db=mnist, batch_size=16) = 13.469
+[HYDRA] 	#2 : db=cifar batch_size=8 lr=0.053 dropout=0.408
+[__main__][INFO] - dummy_training(dropout=0.408, lr=0.053, db=cifar, batch_size=8) = 4.145
+[HYDRA] 	#3 : db=cifar batch_size=8 lr=0.012 dropout=0.305
+[__main__][INFO] - dummy_training(dropout=0.305, lr=0.012, db=cifar, batch_size=8) = 4.133
+[HYDRA] 	#4 : db=mnist batch_size=4 lr=0.030 dropout=0.204
+[__main__][INFO] - dummy_training(dropout=0.204, lr=0.030, db=mnist, batch_size=4) = 1.216
+```
+
+
+and the final 2 evaluations look like this:
+```text
+[HYDRA] 	#8 : db=mnist batch_size=4 lr=0.094 dropout=0.381
+[__main__][INFO] - dummy_training(dropout=0.381, lr=0.094, db=mnist, batch_size=4) = 1.077
+[HYDRA] 	#9 : db=mnist batch_size=4 lr=0.094 dropout=0.381
+[__main__][INFO] - dummy_training(dropout=0.381, lr=0.094, db=mnist, batch_size=4) = 1.077
+[HYDRA] Best parameters: db=mnist batch_size=4 lr=0.094 dropout=0.381
+```
+
+
+The run also creates an `optimization_results.yaml` file in your sweep folder with the parameters recommended by the optimizer:
+```yaml
+best_achieved_result: 0.381
+
+best_parameters:
+  batch_size: 4
+  db: mnist
+  dropout: 0.381
+  lr: 0.094
+
+name: nevergrad:
+```
+
+## Defining the parameters
+
+The plugin can use 3 types of parameters:
+
+### Choices
+
+Choices are defined with **comma-separated values** in the command-line (`db=mnist,cifar` or `batch_size=4,8,12,16`) or with a list in a config file.
+
+**Note:** sequences of increasing scalars are treated as a special case, easier to solve. Make sure to specify it this way when possible.
+
+### Scalars
+Scalars can be defined:
+
+- through a commandline override with **`:`-separated values** defining a range (eg: `dropout=0:1`).
+You can add specifications for log distributed values (eg.: `lr=log:0.001:1`) or integer values (eg.: `batch_size=int:4:8`)
+or a combination of both (eg.: `batch_size=log:int:4:1024`)
+
+- through a config files, with fields:
+  - `init`: optional initial value
+  - `lower` : optional lower bound
+  - `upper`: optional upper bound
+  - `log`: set to `true` for log distributed values
+  - `step`: optional step size for looking for better parameters. In linear mode this is an additive step, in logarithmic mode it
+    is multiplicative. 
+  - `integer`: set to `true` for integers (favor floats over integers whenever possible)
+
+  Providing only `lower` and `upper` bound will set the initial value to the middle of the range, and the step to a sixth of the range.
+
+**Note**: unbounded variables are only available through a config file.
```

Comment:
This is the first time you mention unbounded variables in the doc.
Can you explain what they are before introducing the concept?

Reply:
Is it better now? `unbounded scalars (scalars with no upper and/or lower bounds) can only be defined through a config file.`

Reply:
Much clearer.
You can simplify to it to:
`Scalars with no upper and/or lower bounds can only be defined through a config file.`